bot/seasons/easter/april_fools_vids.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class AprilFoolVideos(commands.Cog):
+    """A cog for april fools that gets a random april fools video from youtube."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.yt_vids = self.load_json()
+        self.youtubers = ['google']  # will add more in future
```

Comment:
Variable created but not used.

Reply:
it will be used once more videos are added by difference youtubers.
currently its only google